lib/api/src/sys/externals/function.rs

```diff
@@ -66,6 +66,44 @@ impl wasmer_types::WasmValueType for Function {
     }
 }
 
+impl Function {
+    /// Convert a `VMFuncRef` into a `Function`.
+    ///
+    /// Returns `None` if the funcref is null.
+    pub fn from_vm_funcref(store: &Store, func_ref: VMFuncRef) -> Option<Self> {
+        if func_ref.is_null() {
+            return None;
+        }
+        let item: &wasmer_vm::VMCallerCheckedAnyfunc = unsafe {
+            let anyfunc: *const wasmer_vm::VMCallerCheckedAnyfunc = *func_ref;
+            &*anyfunc
+        };
+        let export = wasmer_vm::ExportFunction {
+            // TODO:
```

Comment:
Maybe along the unsafety comments add a comment mentioning that this was designed only for examples for now, and not for production usage?

Reply:
“designed” is probably not a right word here, it was intended to be _the_ public API in original wasmer, we just no longer use it as such and its remaining use-case is in test and examples which we hadn't bothered porting away (though could eventually.)